recipes/pycondor/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "PyCondor" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "182d4bf0e35740c7232f00f3a3259369045514666f14ed4c2bd5751940ec1b9c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=2.7,<3.7
+    - pip
+  run:
+    - python >=2.7,<3.7
+    - click
+
+test:
+  requires:
+    - pytest
+  imports:
+    - pycondor
+    - pycondor.job
+    - pycondor.dagman
+    - pycondor.utils
+    - pycondor.basenode
+    - pycondor.cli
+    - click
+
+about:
+  home: https://github.com/jrbourbeau/pycondor
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Python utility for HTCondor'
+  description: |
+    PyCondor (Python HTCondor) is a tool to help build and submit workflows to
+    HTCondor in a straight-forward manner with minimal hassle.
+  doc_url: https://jrbourbeau.github.io/pycondor/
+  dev_url: https://jrbourbeau.github.io/pycondor/
```

Comment:
`dev_url` should be the repository: `dev_url: https://github.com/jrbourbeau/pycondor`